Notifier.php

```diff
@@ -45,6 +45,8 @@ public function sendNewAlerts($period, $idSite)
             $this->sendAlertsPerSmsToRecipient($alerts, new \Piwik\Plugins\MobileMessaging\Model(), $phoneNumber);
         }
 
+        Piwik::postEvent('CustomAlerts.sendNewAlerts', [$triggeredAlerts]);
```

Comment:
Should `$triggeredAlerts` be passed by reference so that event listeners can modify it?
```suggestion
        Piwik::postEvent('CustomAlerts.sendNewAlerts', [&$triggeredAlerts]);
```

Reply:
@snake14 Nothing is being done post this with `$triggeredAlerts`, so didn't pass by reference

Reply:
If the event isn't being used, why is it there? If it's going to be used, shouldn't it allow altering the parameter?

Reply:
@snake14 Its for other plugins Like Slack/Teams to check and sendAlerts